src/components/__tests__/__snapshots__/image-url-input.test.js.snap

```diff
@@ -0,0 +1,44 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<ImageUrlInput /> should match snapshot when Input not visible 1`] = `
+<CSSTransitionGroup
```

Comment:
Do we need this when the input is not visible?

Reply:
Yes, otherwise the animation wouldn't work - this is the component controlling the animations of its children, React docs excerpt:
```
In order for it to apply transitions to its children, the ReactCSSTransitionGroup must already be mounted in the DOM or the prop transitionAppear must be set to true.
```
We can control the appearance by the `transitionAppear` prop but we won't be able to animate when it's going to be removed from the DOM

Reply:
But this might change with `react-transition-group`, I'll introduce it in a subsequent commit